python/GafferImageTest/FormatQueryTest.py

```diff
@@ -88,9 +90,56 @@ def test( self ) :
 			if f.getDisplayWindow().size()[0] * f.getDisplayWindow().size()[1] < 2000 * 2000:
 				self.assertImagesEqual( constantSource["out"], constantDest["out"] )
 
+	def testView( self ) :
+
+		constantSource = GafferImage.Constant()
+		constantSource["color"].setValue( imath.Color4f( 0.1, 0.2, 0.3, 0.4 ) )
+		constantSource["format"].setValue( GafferImage.Format( imath.Box2i( imath.V2i( 0 ), imath.V2i( 512 ) ), 1 ) )
+
+		reader = GafferImage.ImageReader()
+		reader["fileName"].setValue( os.path.expandvars( "$GAFFER_ROOT/python/GafferImageTest/images/checkerboard.100x100.exr" ) )
+
+		views = GafferImage.CreateViews()
+		views["views"].addChild( Gaffer.NameValuePlug( "left", GafferImage.ImagePlug(), True ) )
+		views["views"].addChild( Gaffer.NameValuePlug( "right", GafferImage.ImagePlug(), True ) )
+		views["views"][0]["value"].setInput( constantSource["out"] )
+		views["views"][1]["value"].setInput( reader["out"] )
+
+		formatQuery = GafferImage.FormatQuery()
+		formatQuery["image"].setInput( views["out"] )
+
+		# The result when contextView and queryView are both not overridden is technically undefined behaviour
+		# - views has just left and right views, so it's illegal to ask for "default".  But we know in practice
+		# that CreateViews will currently return the default value of an ImagePlug, which has an HD format
```

Comment:
This comment seems to contradict the `assertRaisesRegex()` at L127. Seems like the assertion is correct and the comment is wrong?